modules/swagger-codegen-cli/pom.xml

```diff
@@ -18,6 +18,7 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
+                <filtering>true</filtering>
```

Comment:
Unless I'm missing anything build-wise, I think this should not be a problem because the filtering is enabled just for the `swagger-codegen-cli` artifact and should not apply to the resources from `swagger-codegen` artifact (which is where all the templates are included as resources).

Running the samples should be a good practice anyway, so I'll try to find the time to do so, however.